### PR TITLE
communicate group rates

### PR DESCRIPTION
**Pull request by totto82**
Tested on the spe9group case. 

We may want to reduce the number of communication points to speed up the run.

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by alfbr** [comment]
If this is rebased, I can start testing it. A merge would be even better :-)

File: opm/simulators/wells/BlackoilWellModel_impl.hpp [hunk 1]
```diff
@@ -391,8 +420,10 @@ namespace Opm {
                 : 0.0;
             guideRate_->compute(well->name(), reportStepIdx, simulationTime, oilpot, gaspot, waterpot);
```
Comment (atgeirr) [inline opm/simulators/wells/BlackoilWellModel_impl.hpp]: Should not this block be removed, replaced by the above block?

**Comment by totto82** [comment]
Rebased, comments addressed and test added.

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
jenkins build this please

**Pull request merged**